src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -1,9 +1,182 @@
-import { Text } from "@artsy/palette"
+import {
+  Box,
+  Button,
+  Checkbox,
+  Column,
+  Flex,
+  Separator,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { useState } from "react"
 
 export const PreferencesApp: React.FC = () => {
+  const [uncheckedAll, setUncheckedAll] = useState(false)
+
   return (
     <>
-      <Text variant="xl">Preferences App</Text>
+      <Text variant="xl" mt={4}>
+        Preferences Center
+      </Text>
+
+      <Separator mt={2} />
+
+      <Column
+        span={12}
+        pb={2}
+        pr={12}
```

Comment:
We should always stay within our design system [spacing scale](https://palette.artsy.net/atoms/spacing/#all-spacing-values). The only valid options for margin / padding are `0.5, 1, 2, 4, 6, 12`. Anything outside of these means that we've left the design system theme and design drift starts to occur. 

If yr ever curious about all the possible theme values have a look at [our theme](https://github.com/artsy/palette/blob/main/packages/palette-tokens/src/themes/v3.tsx)

Reply:
(But in addition: You would almost never have a need to use padding on a `Column` since you'd control spacing with the `gridRowGap` or the `gridColGap`)

Reply:
Ah yup, using the grid simplifies layout work considerably and makes one not need to think about any of this. It's just, "how many columns do I need out of the 12? Two columns? `span={2}`